packages/features/bookings/BookingMultiFiltersStore.ts

```diff
@@ -0,0 +1,52 @@
+import { create } from "zustand";
+
+import type { SVGComponent } from "@calcom/types/SVGComponent";
+import { User, Link, MapPin } from "@calcom/ui/components/icon";
+
+interface addFilterOption {
+  StartIcon?: SVGComponent;
+  label: "people" | "event_type" | "date_range" | "location";
+}
+
+export type BookingMultiFilterStore = {
+  addFilterOptions: addFilterOption[];
+  toggleOption: (option: addFilterOption) => void;
+  isFilterActive: (label: addFilterOption["label"]) => boolean;
+};
+
+export const useBookingMultiFilterStore = create<BookingMultiFilterStore>((set, get) => ({
+  addFilterOptions: [
+    {
+      label: "people",
+      StartIcon: User,
+    },
+    {
+      label: "event_type",
+      StartIcon: Link,
+    },
+    // {
+    //   label: "date_range",
+    //   StartIcon: Calendar,
+    // },
```

Comment:
Filter by date range will be added later

Reply:
You can reuse the component I used on insights 👍